alicloud/resource_alicloud_ess_scalingconfiguration.go

```diff
@@ -431,6 +439,11 @@ func modifyEssScalingConfiguration(d *schema.ResourceData, meta interface{}) err
 		d.SetPartial("internet_max_bandwidth_out")
 	}
 
+	if d.HasChange("credit_specification") {
+		request.SystemDiskCategory = d.Get("credit_specification").(string)
```

Comment:
I think it should be `request.CreditSpecification = d.Get("credit_specification").(string)`

Reply:
Yeah, my bad. Fixed now.